Gemfile

```diff
@@ -6,7 +6,7 @@ ruby '3.2.2'
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
 gem 'rails', '~> 7.0.7'
 gem 'rubocop', '>= 1.0', '< 2.0'
-
+gem 'web-console'
```

Comment:
- [ ] When I try setting up your app, I get this error from the Gemfile due to this gem. This gem should only be in the development environment, but it available in the test environment as well. Kindly fix this by moving it to the development group.

  ![image](https://github.com/StevenWafeek/blog-app/assets/54848363/ce68a030-4d6b-4be9-b386-e3edb6a24d97)

  ```
  group :development do
    ...
    ...
    gem 'web-console'
  end
  ```